sklearn/ensemble/_bagging.py

```diff
@@ -779,8 +780,7 @@ def predict_log_proba(self, X):
 
             return log_proba
 
-        else:
-            return np.log(self.predict_proba(X))
+        return np.log(self.predict_proba(X))
```

Comment:
As mentioned elsewhere, I think this hurts readability. What do we gain from removing the `else`?

This will trick the inattentive reader (aka me) into thinking that this function only every returns `np.log(self.predict_proba(X))`